cluster/examples/kubernetes/monitoring/tectonic-service-monitor.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: rook
+  namespace: rook
+  labels:
+    cluster: rook
+spec:
+  namespaceSelector:
+    matchNames:
+    - rook
```

Comment:
Could you please add the `namespaceSelector` to the https://raw.githubusercontent.com/rook/rook/master/cluster/examples/kubernetes/monitoring/service-monitor.yaml file